contracts/market-wrapper/NounsMarketWrapper.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.5;
+
+// ============ External Imports ============
+import {INounsAuctionHouse} from "../external/interfaces/INounsAuctionHouse.sol";
+
+// ============ Internal Imports ============
+import {IMarketWrapper} from "./IMarketWrapper.sol";
+
+/**
+ * @title NounsMarketWrapper
+ * @author Anna Carroll + Nounders
+ * @notice MarketWrapper contract implementing IMarketWrapper interface
+ * according to the logic of Nouns' Auction Houses
+ */
+contract NounsMarketWrapper is IMarketWrapper {
+    // ============ Internal Immutables ============
+
+    INounsAuctionHouse internal immutable market;
+
+    // ======== Constructor =========
+
+    constructor(address _nounsAuctionHouse) {
+        market = INounsAuctionHouse(_nounsAuctionHouse);
+    }
+
+    // ======== External Functions =========
+
+    /**
+     * @notice Determine whether there is an existing auction
+     * for this token is active
+     * @return TRUE if the auction exists
+     */
+    function auctionExists(uint256 auctionId)
```

Comment:
can we add a comment about how `currentAuctionId` = `tokenId` in the nouns auction house world?